ui.apps/src/main/content/jcr_root/apps/etoolbox-backpack/assets/clientlibs/backpackcoral/css/styles.css

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+[data-globalnav-anchor-href='/tools/etoolbox/backpack.html'] .coral3-Icon--gift:before {
+    content: url("../../../backpack-icon.png");
```

Comment:
Not an issue now, but consider swapping this for an SVG icon in the future. 
SVG is:
 - better renderable because is vector, and all the UI graphics in AEM are simplistic and monochrome => walk and talk like natural-born vector graphics;
 - can be put inline and will travel with the CSS wherever it goes; while the current link may break one day
 - can be styled itself, like: different fill colors, etc.

Reply:
thanks, I've changed and added for sites panel